app/src/components/view/ranking/RankingListToday.tsx

```diff
@@ -63,13 +63,26 @@ const RankingListToday: React.FC<{ selectedTopic: number }> = ({
 			}
 		};
 
-		fetchData(getDate());
+		if (selectedTopic !== 0) {
+			fetchData(getDate());
+		}
```

Comment:
トピックが0の時（今日のお題がない時）、ローディングが消えないので、対応お願いします
```suggestion
		if (selectedTopic !== 0) {
			fetchData(getDate());
		}
		setIsLoading(false);
```